include/ppconsul/kv.h

```diff
@@ -63,9 +182,19 @@ namespace ppconsul { namespace kv {
         namespace groups {
             KWARGS_KEYWORDS_GROUP(get, (consistency, dc, block_for, token))
             KWARGS_KEYWORDS_GROUP(put, (flags, token, dc))
+            KWARGS_KEYWORDS_GROUP(txn, (consistency, token, dc))
         }
     }
 
+    namespace impl {
+        StringList parseKeys(const std::string& resp);
+        std::vector<KeyValue> parseValues(const std::string& resp);
+
+        std::vector<KeyValue> txnParseValues(const std::string& resp);
+        TxnAborted txnParseErrors(const std::string& resp);
```

Comment:
I'd change this to return `std::vector<TxnError>` instead of exception object itself. Mostly to make throw line to be more obvious `throw TxnAborted(impl::txnParseErrors(...))`.

Reply:
> Btw, do you have plane to make dedicated Tx endpoint or not?

As for now, no. We only need KV transactions for our job, and it’s quite unlikely we are going to need the other ones, but — who knows :)